css/src/components/button.scss

```diff
@@ -0,0 +1,324 @@
+@use 'sass:map';
+
+$button-color: $text !default;
+$button-background-color: $body-background !default;
+
+$button-large-font-size: $font-size-6 !default;
+$button-small-font-size: $font-size-8 !default;
+$button-x-small-font-size: $font-size-9 !default;
+
+$button-border-color: $text-subtle !default;
+$button-border-width: $control-border-width !default;
+$button-border-radius: $radius-small;
+
+$button-padding-vertical: calc(0.375em - #{$button-border-width}) !default;
+$button-padding-horizontal: 0.75em !default;
+
```

Comment:
Verified buttons sizes remain equivalent to the existing buttons in docs-ui. Not a requirement, but was curious.
```js
console.table(Array.from(document.querySelectorAll('.input,.button')).map(({ offsetHeight, className }) => ({ offsetHeight, className })))
```